tests/wallet/inscribe.rs

```diff
@@ -1343,3 +1343,86 @@ fn inscriptions_are_not_compressed_if_no_space_is_saved_by_compression() {
   assert_eq!(response.status(), StatusCode::OK);
   assert_eq!(response.text().unwrap(), "foo");
 }
+
+#[test]
+fn batch_inscribe_fails_if_invalid_destination_address() {
```

Comment:
I think we can remove this. But we should have a test that if the network is wrong, we catch it.